types/index.d.ts

```diff
@@ -78,7 +78,7 @@ export class Canvas {
 	/** Constant used in PNG encoding methods. */
 	readonly PNG_FILTER_PAETH: number
 
-	constructor(width: number, height: number, type?: 'pdf'|'svg')
+	constructor(width: number, height: number, type?: 'pdf'|'svg'|'jpg')
```

Comment:
`'jpg'` isn't a valid Canvas type, just `'pdf'` and `'svg'`. Anything else is ignored and creates an image canvas. If you want you could add `'image'` here to maybe improve clarity...

https://github.com/Automattic/node-canvas/blob/37d556c3cd9704d8acc0071d991e4bad4f69e10e/src/Canvas.cc#L105-L110

Reply:
Ahh thanks for that clarity! Updated